### PR TITLE
Amplify init

### PR DESCRIPTION
**Pull request by tomas90hampl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +28,23 @@ yarn-error.log*
 
 # misc
 .DS_Store
+
+#amplify
```
Comment (macajind) [inline .gitignore:32]: Space :grin:
```suggestion
# amplify
```

File: .gitignore [hunk 2]
```diff
@@ -16,6 +16,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+src/config/env/.env.dev
```
Comment (macajind) [inline .gitignore:19]: You don't need to put explicit path here, just file name should be fine. :wink: 

And I would use config files that are already here e.g. `.env.development.local` rather then creating new ones. :sweat_smile:

File: src/config/index.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const configuration = {
+	aws_project_region: '',
+	aws_cognito_identity_pool_id: '',
+	aws_cognito_region: '',
+	aws_user_pools_id: '',
+	aws_user_pools_web_client_id: '',
+	oauth: '',
+}
+
+export default configuration
```
Comment (macajind) [inline src/config/index.ts:10]: This is useless. There needs to be process how to populate this configuration from AWS. Let's try it together later. :wink:

**Review by macajind — CHANGES_REQUESTED** [review]
What configuration did you choose for Amplify Auth? Email with default settings or something else? :thinking:

File: .gitignore (hunk 1 above)
Comment (tomas90hampl) [reply .gitignore:32]: :+1:

File: .gitignore (hunk 2 above)
Comment (tomas90hampl) [reply .gitignore:19]: Ok :+1: :wink:

**Comment by tomas90hampl** [comment]
> What configuration did you choose for Amplify Auth? Email with default settings or something else?

I choose Email with default settings

File: src/config/index.ts (hunk 3 above)
Comment (tomas90hampl) [reply src/config/index.ts:10]: Ok, I delete the configuration settings for now. Due to pipeline. I yand import configuration from gitignored json settings

**Review by macajind — APPROVED** [review]
Good job! :1st_place_medal:

**Pull request merged**